core/dbt/task/runnable.py

```diff
@@ -154,7 +154,7 @@ def get_node_selector(self) -> NodeSelector:
 
     @abstractmethod
     def defer_to_manifest(self, adapter, selected_uids: AbstractSet[str]):
-        raise NotImplementedException(f"defer_to_manifest not implemented for task {type(self)}")
+        raise NotImplementedError(f"defer_to_manifest not implemented for task {type(self)}")
```

Comment:
@emmyoop - Noticed this import error via [mypy from the latest changes](https://github.com/dbt-labs/dbt-core/actions/runs/3888238595/jobs/6635350189), addressed it here as part of my merge.

Reply:
thanks.  no idea how i missed that.